app/views/idv/review/new.html.erb

```diff
@@ -38,11 +33,6 @@
   <div class="text-right margin-top-neg-4 margin-bottom-4">
     <%= link_to(t('idv.forgot_password.link_text'), idv_forgot_password_url, class: 'margin-left-1') %>
   </div>
-  <%= render AccordionComponent.new do |c| %>
-    <% c.with_header { t('idv.messages.review.intro') } %>
-    <%= render 'shared/pii_review', pii: @applicant,
```

Comment:
I'd thought this was the only place this `pii_review` partial is used, but I'd forgot it's also shown on the screen prompting the user for their password after reactivating their account. Will we want to remove it there as well, since its purpose is very similar?

![image](https://github.com/18F/identity-idp/assets/1779930/3b6d021c-70a0-4146-9d18-8e580464da48)

Reply:
Good catch -- let me check with the team

Reply:
@eric-gade - This is splitting to https://cm-jira.usa.gov/browse/LG-9974 per a conversation I had with @aduth.

Reply:
Follow-up pull request for LG-9974 at #8532